### PR TITLE
feat: popout window

### PR DESCRIPTION
**Pull request by emilbayes**

### CODE REVIEW

File: web-extension/backend/admin-ns.js [hunk 1]
```diff
@@ -23,9 +26,36 @@ function doValidate(validator, params) {
  * @returns {JSONRPCServer}
  */
 export default function init({ encryptedStore, settings, wallets, networks, connections, onerror }) {
+  let popout = null
+  windows.onRemoved.addListener(() => {
```
Comment (emilbayes) [inline web-extension/backend/admin-ns.js:30]: @dexturr should we perhaps check the `windowId` here just to be absolutely sure we are setting the popout to null in the right cases? https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/windows/onRemoved#parameters

Comment (emilbayes) [reply web-extension/backend/admin-ns.js:38]: Ah this breaks BE tests. We can fudge it a bit by moving this listener into the setup of the popup and remove it on close. I'll do that monday

Comment (dexturr) [reply web-extension/backend/admin-ns.js]: Have made the change for the windowId stuff

**Comment by dalebennett1992** [comment]
Ok so this still needs a unit test to check the pop out has been brought into focus, otherwise it is just the visual that is missing. Also I think we should investigate why I get 3 scrollers when I hit the pop out haha

**Review by dalebennett1992 — APPROVED** [review]

**Pull request merged**